playground.html

```diff
@@ -635,9 +635,9 @@
             <wb-accordion class="vector">
                 <header>
                     <img class="cat-icon" src="images/icon/vector.svg">
-                    Vectors
+                    Points & Vectors
                 </header>
-                <p>Waterbear vectors are 2-D objects expressing x and y coordinates, but also a direction and magnitude (useful for speed, acceleration, distance).</p>
+                <p>Waterbear points and vectors are x,y coordinates but vectors also have a direction and magnitude (useful for speed, acceleration, distance).</p>
```

Comment:
"Waterbear vectors are x,y coordinates that can be used as points, but also have…"